app/domain/errors.rb

```diff
@@ -273,6 +273,18 @@ module AuthnK8s
           msg:  "Invalid Kubernetes host id: {0}. Must end with <namespace>/<resource_type>/<resource_id>",
           code: "CONJ00048E"
         )
+
+        CSRMissingCNEntry = ::Util::TrackableErrorClass.new(
+            msg:  "CSR [subject: {0}, spiffe_id: {1} must have a CN entry."+
+                " Account: {2}, Service Name: {3}",
```

Comment:
you have an opening `[` without closing it. 
why do you print the account and service name? we don't do it in other messages and they are already visible in the logs. I would print only the subject and spiffe-id, if they are not sensitive.

Please change to `CSR with subject {0-subject} and spiffe-id {1-spiffe-id} must have a CN entry`

Reply:
I did it and it did not print all the params, I don't know why when I switched to plain numbers it worked

Reply:
it should work. what did you write that didn't work for you?